frontend/apps/reader/modules/readertoc.lua

```diff
@@ -698,7 +700,7 @@ function ReaderToc:onShowToc()
         align_baselines = true,
         items_per_page = items_per_page,
         items_font_size = items_font_size,
-        line_color = require("ffi/blitbuffer").COLOR_WHITE,
+        line_color = items_show_separator and Blitbuffer.COLOR_DARK_GRAY or Blitbuffer.COLOR_WHITE,
```

Comment:
Haven't checked, but I assume it's consistent with Menu's choice or color for the separator? ;).

Reply:
https://github.com/koreader/koreader/blob/ea3fa5c2c7ed7330e8e1e6ef838b780e7998ea51/frontend/ui/widget/menu.lua#L582